editoast/src/views/projection.rs

```diff
@@ -32,15 +32,17 @@ use crate::core::simulation::SignalCriticalPosition;
 use crate::core::simulation::SimulationResponse;
 use crate::core::simulation::ZoneUpdate;
 use crate::error::Result;
+use crate::models;
 use crate::models::RollingStockModel;
 use crate::models::infra::Infra;
 use crate::models::prelude::*;
 use crate::models::train_schedule::TrainSchedule;
 use crate::views::path::pathfinding::PathfindingResult;
 use crate::views::path::projection::PathProjection;
 use crate::views::path::projection::TrackLocationFromPath;
-use crate::views::train_schedule::TrainScheduleError;
-use crate::views::train_schedule::train_simulation_batch;
+
+use super::timetable::train_schedule::TrainScheduleError;
```

Comment:
nitpick: all other imports use `crate` but this one use `super`. Probably use `crate` everywhere which is less contextual (kind of absolute vs relative).